backends/tfhe-cuda-backend/cuda/include/device.h

```diff
@@ -11,75 +11,78 @@
 
 extern "C" {
 
+#define check_cuda_error(ans)                                                  \
+  { cuda_error((ans), __FILE__, __LINE__); }
+inline void cuda_error(cudaError_t code, const char *file, int line,
+                       bool abort = true) {
```

Comment:
any reason to have abort as a flag here ? looks like we are not forwarding anything to it in the macro ?

Reply:
We do:
```
  if (code != cudaSuccess) {
    fprintf(stderr, "Cuda error: %s %s %d\n", cudaGetErrorString(code), file,
            line);
    if (abort)
      exit(code);
  }
```
We could remove the abort bool argument and just call abort() in the macro, is that what you mean?

Reply:
yep, I'm feeling like the abort flag might not be useful here ?

Reply:
it might be use I don't know :)

Reply:
I just removed it, assuming any time we encounter a CudaError we want to abort.